addons/sale/views/sale_portal_templates.xml

```diff
@@ -481,68 +481,71 @@
                         <t t-set="current_subtotal" t-value="0"/>
 
                         <t t-foreach="sale_order.order_line" t-as="line">
+                            <t t-if="not line._downpayment_state() and line._show_downpayment_section()">
```

Comment:
```suggestion
                        <t t-set="order_lines_to_report" t-value="order._get_order_lines_to_report()"/>
                        <t t-foreach="order_lines_to_report" t-as="line">
```
Avoid modifying the whole report template when not needed.
Furthermore, if you correctly write `_get_order_lines_to_report`, the returned lines won't contain the down payment lines and sections, and therefore that's all you would have to change in the report imho.